src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = century
+    ? people.filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
+
+  const averageAge = men
+    .reduce((a, man) => a + (man.died - man.born), 0)
+    / men.length;
+
+  return averageAge;
```

Comment:
I think this const is redundant
```suggestion
 return men
    .reduce((a, man) => a + (man.died - man.born), 0)
    / men.length;
```